mlir/test/Conversion/VectorToSPIRV/vector-to-spirv.mlir

```diff
@@ -613,6 +613,17 @@ func.func @shuffle(%v0 : vector<4xi32>, %v1: vector<4xi32>) -> vector<1xi32> {
 
 // -----
 
+// CHECK-LABEL:  func @shuffle
+//  CHECK-SAME:  %[[ARG0:.+]]: vector<4xi32>, %[[ARG1:.+]]: vector<4xi32>
+//       CHECK:    %[[SHUFFLE:.*]] = spirv.VectorShuffle [1 : i32, -1 : i32, 5 : i32, -1 : i32] %[[ARG0]], %[[ARG1]] : vector<4xi32>, vector<4xi32> -> vector<4xi32>
```

Comment:
SPIR-V requires poison elements to be 0xFFFFFFFF instead of -1: https://registry.khronos.org/SPIR-V/specs/unified1/SPIRV.html#:~:text=A%20Component%20literal%20may%20also%20be%20FFFFFFFF%2C%20which%20means%20the%20corresponding%20result%20component%20has%20no%20source%20and%20is%20undefined.%20All%20Component%20literals%20must%20either%20be%20FFFFFFFF%20or%20in%20%5B0%2C%20N%20%2D%201%5D . Because the indices are `I32ArrayAttr` in MLIR, I don't think we need to do anything here, despite the textual format looking as if this is incorrect.